CordovaExamples/www/js/index.js

```diff
@@ -204,17 +239,34 @@ var cordovaExample = {
     });
 
     marker = new google.maps.Marker({
-      position : new google.maps.LatLng(65.060848804763, 25.4410770535469),
-      map : venuemap,
-      icon : image,
-      zIndex : google.maps.Marker.MAX_ZINDEX + 1,
-      optimized : false
+      position: new google.maps.LatLng(65.060848804763, 25.4410770535469),
+      map: venuemap,
+      icon: image,
+      zIndex: google.maps.Marker.MAX_ZINDEX + 1,
+      optimized: false
     });
 
     marker.setVisible(false);
     marker.setMap(venuemap);
     accuracyCircle.setVisible(false);
     accuracyCircle.setMap(venuemap);
+
+    readJsonAsset('www/data/wayfinding-graph.json', function (graph) {
+        buildWayfindingController(graph, venuemap).then(function(wfc) {
```

Comment:
Hmm. I think you should check here if `!graph` (graph is an empty string / null) and then not try to build the controller